ui/app/clinical/dashboard/controllers/patientDashboardController.js

```diff
@@ -13,9 +13,36 @@ angular.module('bahmni.clinical')
             $scope.visitSummary = visitSummary;
             $scope.enrollment = $stateParams.enrollment;
             $scope.isDashboardPrinting = false;
+            $scope.loadIPD = false;
             var programConfig = appService.getAppDescriptor().getConfigValue("program") || {};
             $state.discardChanges = false;
 
+            $scope.alergyData = {
```

Comment:
@AshishkurianTw @Phanindra-tw @tanyaa-tw I assume the changes made here will be revised later on. I believe this data was added for testing the micro-frontend setup. Please confirm, as everything else appears fine to me.

Reply:
Yes @rahu1ramesh. This is a temporary addition . The upcoming PRs will update these values.